frontend/src/metabase/common/components/AggregationPicker/AggregationPicker.tsx

```diff
@@ -365,6 +377,38 @@ function renderItemName(item: ListItem) {
   return item.displayName;
 }
 
+function renderItemWrapper(content: ReactNode) {
+  return <HoverParent>{content}</HoverParent>;
+}
+
+function renderItemIcon(item: ListItem) {
+  if (item.type !== "metric") {
+    return null;
+  }
+
+  if (!item.description) {
+    return null;
+  }
+
+  return (
+    <Flex pr="sm" align="center">
+      <Popover
+        position="right"
+        content={
+          <Box p="md">
+            <Markdown>{item.description}</Markdown>
```

Comment:
We should probably use `disallowHeading` and `unstyleLinks` as we do in similar places https://github.com/metabase/metabase/blob/dfab26710da2b14bf1a1b43c4e941f396d3f9ec6/frontend/src/metabase/visualizations/components/ScalarValue/ScalarValue.jsx#L105. Could you confirm this with the product? Could you add tests for these cases?

Reply:
@mngr Can you confirm we want to render this as Markdown without headings and with unstyled links? We render de description as such elsewhere so I think it makes sense.

Reply:
> @mngr Can you confirm we want to render this as Markdown without headings and with unstyled links? We render de description as such elsewhere so I think it makes sense.

Yes, let's do that